desktop/containerd/index.md

```diff
@@ -32,186 +32,100 @@ images** checkbox.
+$ docker buildx build --quiet \
+  --platform=linux/amd64,linux/arm64 \
+  --builder=objective_blackburn \
+  --load .
+ERROR: docker exporter does not currently support exporting manifest lists
 ```
 
-Run multi-platform image:
+Enabling the containerd image store lets you build, and load, multi-platform images
+to your local image store, all while using the default builder.
 
 ```console
-$ docker run <username>/hello-friends
-
+$ docker info | grep "Storage Driver" -A1 
+ Storage Driver: stargz
+  driver-type: io.containerd.snapshotter.v1
+$ docker buildx build \
+  --platform=linux/amd64,linux/arm64 \
+  --tag=user/containerd-multiplatform .
+[+] Building 6.5s (11/11) FINISHED
+ => [internal] load .dockerignore
+ => => transferring context: 2B
+ => [internal] load build definition from Dockerfile
+ => => transferring dockerfile: 115B
+ => resolve image config for docker.io/docker/dockerfile:1
+ => [auth] docker/dockerfile:pull token for registry-1.docker.io
+ => docker-image://docker.io/docker/dockerfile:1@sha256:39b85bbfa7536a5feceb7372a0817649ecb2724562a38360f4d6a7782a409b14
+ => => resolve docker.io/docker/dockerfile:1@sha256:39b85bbfa7536a5feceb7372a0817649ecb2724562a38360f4d6a7782a409b14
+ => => sha256:9d0cd65540a143ce38aa0be7c5e9efeed30d3580d03667f107cd76354f2bee65 3.15MB / 10.82MB
+ => => extracting sha256:9d0cd65540a143ce38aa0be7c5e9efeed30d3580d03667f107cd76354f2bee65
+ => [linux/arm64 internal] load metadata for docker.io/library/alpine:latest
+ => [linux/amd64 internal] load metadata for docker.io/library/alpine:latest
+ => [auth] library/alpine:pull token for registry-1.docker.io
+ => [linux/arm64 1/1] FROM docker.io/library/alpine@sha256:124c7d2707904eea7431fffe91522a01e5a861a624ee31d03372cc1d138a3126
+ => => resolve docker.io/library/alpine@sha256:124c7d2707904eea7431fffe91522a01e5a861a624ee31d03372cc1d138a3126
+ => => sha256:c41833b44d910632b415cd89a9cdaa4d62c9725dc56c99a7ddadafd6719960f9 0B / 3.26MB
+ => [linux/amd64 1/1] FROM docker.io/library/alpine@sha256:124c7d2707904eea7431fffe91522a01e5a861a624ee31d03372cc1d138a3126
+ => => resolve docker.io/library/alpine@sha256:124c7d2707904eea7431fffe91522a01e5a861a624ee31d03372cc1d138a3126
+ => => sha256:f56be85fc22e46face30e2c3de3f7fe7c15f8fd7c4e5add29d7f64b87abdaa09 73.73kB / 3.37MB
+ => exporting to image
+ => => exporting layers
+ => => exporting manifest sha256:d28bdb47b683d725f5b2e8a152f94b70a429ec895c221f5c8fad4474eeaf2f2f
+ => => exporting config sha256:878d877e4f705ab32e83c1334412d2f19c6d9749cf5dcb44c80d059ea95a4ff7
+ => => exporting manifest sha256:ee79e74f9211d6e4d86a66e849b72f740c38e3ee76524dda8e8b819909e36b8d
+ => => exporting config sha256:a579f49700dcc9e7f354769ceb34ffd77264ff66d18f225929e6cbe51053e3ed
+ => => exporting manifest list sha256:7401bb14c229a1f06703763fdfe3a2dd70dbf083e678d5b11c4063aed92551b9
+ => => naming to docker.io/library/user/containerd-multiplatform:latest
+ => => unpacking to docker.io/library/user/containerd-multiplatform:latest
```

Comment:
Perhaps we can

- remove some redundant lines (to put more focus on the important ones
- (maybe personal preference): I do like showing the `time` at the end of the lines; they format nicely to show durations (although I guess the old example was "from cache").
- ^^ that said, I may ellipsise some of the longer lines (digests)

E.g., taking the original example, and if the output is too wide, I'd format it something like;

```
[+] Building 0.7s (7/7)
 FINISHED

  => [internal] load .dockerignore                                                       0.0s
  => => transferring context: 2B                                                         0.0s
  => [internal] load build definition from Dockerfile                                    0.0s
  => => transferring dockerfile: 88B                                                     0.0s
  => [linux/arm64 internal] load metadata for docker.io/library/alpine:latest            0.6s
  => [linux/amd64 internal] load metadata for docker.io/library/alpine:latest            0.6s
  => [linux/amd64 1/1] FROM docker.io/library/alpine@sha256:bc41182d7ef5ffc53a40...      0.0s
  => => resolve docker.io/library/alpine@sha256:bc41182d7ef5ffc53a40b044e725193b...      0.0s
  => CACHED [linux/arm64 1/1] FROM docker.io/library/alpine@sha256:bc41182d7ef5f...      0.0s
  => => resolve docker.io/library/alpine@sha256:bc41182d7ef5ffc53a40b044e725193b...      0.0s
  => exporting to image                                                                  0.0s
  => => exporting layers                                                                 0.0s
  => => exporting manifest sha256:71bf02afcd7a791c268aa935027f1dc05238f5b5017d75...      0.0s
  => => exporting config sha256:f1edbf6b99d22831f9312ab2b8b7642a904c614000bb8369...      0.0s
  => => exporting manifest sha256:e41da7320a956163128d77ad69c8109af4799b41bd2e8e...      0.0s
  => => exporting config sha256:4a5580ab8335432cf6cea7ff695f177d120fa2c9aa400252...      0.0s
  => => exporting manifest list sha256:339d089b539c950268de1edeef9652584e16efa51...      0.0s
  => => naming to docker.io/<username>/hello-friends:latest                              0.0s
  => => unpacking to docker.io/<username>/hello-friends:latest
```

And

- removing (mostly irrelevant) lines
- keeping some that shows that it's;
    - transferring the Dockerfile
    -  building _multi architectures_ (which is the relevant information here)
    - "naming  to", and "unpacking to", which show that the created image is tagged with the name the user provided

```
[+] Building 0.7s (7/7)
 FINISHED
  ...
  => => transferring dockerfile: 88B                                                     0.0s
  => [linux/arm64 internal] load metadata for docker.io/library/alpine:latest            0.6s
  => [linux/amd64 internal] load metadata for docker.io/library/alpine:latest            0.6s
  => [linux/amd64 1/1] FROM docker.io/library/alpine@sha256:bc41182d7ef5ffc53a40...      0.0s
  => => resolve docker.io/library/alpine@sha256:bc41182d7ef5ffc53a40b044e725193b...      0.0s
  => [linux/arm64 1/1] FROM docker.io/library/alpine@sha256:bc41182d7ef5ffc53a40...      0.0s
  => => resolve docker.io/library/alpine@sha256:bc41182d7ef5ffc53a40b044e725193b...      0.0s
  => exporting to image                                                                  0.0s
  => => exporting layers                                                                 0.0s
  ...
  => => naming to docker.io/<username>/hello-friends:latest                              0.0s
  => => unpacking to docker.io/<username>/hello-friends:latest
```

Reply:
Agree that it's too verbose now. When I initially drafted it, I actually ran the command with the `--quiet` flag, since the output is mostly noise. But the quiet output only shows a SHA256 digest, so it's a bit too cryptic even for me.

I'll add a truncated version!